src/cmd/dev.go

```diff
@@ -25,26 +24,30 @@ var devDeployCmd = &cobra.Command{
 	Args:  cobra.MaximumNArgs(1),
 	Short: lang.CmdDevDeployShort,
 	Long:  lang.CmdDevDeployLong,
-	PreRun: func(_ *cobra.Command, args []string) {
-		setBundleFile(args)
-	},
 	Run: func(_ *cobra.Command, args []string) {
+		config.Dev = true
 
-		// Create Bundle
-		srcDir, err := os.Getwd()
-		if err != nil {
-			message.Fatalf(err, "error reading the current working directory")
-		}
+		// Get bundle source
+		src := ""
 		if len(args) > 0 {
-			srcDir = args[0]
+			src = args[0]
 		}
 
-		if len(srcDir) != 0 && srcDir[len(srcDir)-1] != '/' {
-			srcDir = srcDir + "/"
-		}
+		// Check if source is a local bundle
+		localBundle := helpers.IsDir(src)
+
+		if localBundle {
+			// Create Bundle
+			setBundleFile(args)
+
+			// Ensure source ends with a slash
+			if len(src) != 0 && src[len(src)-1] != '/' {
```

Comment:
Why do we need the `/` at the end? (could also use `strings.HasSuffix` here)

Reply:
deleted.... was a band-aid for another bug. 🐛